rust-runtime/aws-smithy-client/README.md

```diff
@@ -0,0 +1,7 @@
+# aws-smithy-client
+
+`aws-smithy-client` defines a Tower-based client that implements functionality that exists across all smithy-based service clients including:
```

Comment:
I think a lot of these readme descriptions answer "what is this?" if the context of what smithy-rs and aws-sdk-rust are is already established, but if someone just stumbled upon these packages in crates.io, they would have no way of knowing they're part of the AWS SDK or a part of smithy-rs apart from reading the package prefix, and also no way of knowing that they're not intended to be consumed directly.

I think a boilerplate sentence in them could go a long way for that.

Reply:
great idea, done! Let me know if you have any suggestions about the specific sentence contents and I'll push an update